docs/VictoriaLogs/data-ingestion/Logstash.md

```diff
@@ -1,5 +1,10 @@
 # Logstash setup
 
+[Logstash](https://www.elastic.co/guide/en/logstash/8.8/introduction.html) log collector supports
+[Elasticsearch output](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html) compatible with
```

Comment:
Logstash uses opensearch plugin for output, maybe we should use reference to this plugin?
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/victorialogs/logstash/pipeline.conf

Here is a readme for it which describes why opensearch plugin is used - https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/victorialogs/logstash/README.md

Reply:
fixed